src/core/src/lib.rs

```diff
@@ -117,6 +117,8 @@ pub enum ShaderSet<R: Resources> {
     Geometry(VertexShader<R>, GeometryShader<R>, PixelShader<R>),
     /// Tessellated TODO: Tessellated, TessellatedGeometry, TransformFeedback
     Tessellated(VertexShader<R>, HullShader<R>, DomainShader<R>, PixelShader<R>),
+    /// Tessellated TODO: Tessellated, TessellatedGeometry, TransformFeedback
```

Comment:
let's remove the TODO from here and above comment

Reply:
Done